frontend/src/container/Dashboard.jsx

```diff
@@ -88,7 +85,9 @@ export class Dashboard extends React.Component {
     ])
 
     props.registerLiveMessageHandlerList([
-      { entityType: TLM_ET.SHAREDSPACE, coreEntityType: TLM_CET.MODIFIED, handler: this.handleWorkspaceModified }
+      { entityType: TLM_ET.SHAREDSPACE, coreEntityType: TLM_CET.MODIFIED, handler: this.handleWorkspaceModified },
+      { entityType: TLM_ET.SHAREDSPACE_SUBSCRIPTION, coreEntityType: TLM_CET.CREATED, handler: this.handleNewRequest },
```

Comment:
is there a naming difference between "request" and "subscription"?

Reply:
No, it's a "subscription request"

Reply:
ok, then keeping the wording "subscription" would be more consistent I guess, unless we usually use "request" in the frontend but I don't think so (not sure though)

Reply:
fix